ops/jenkins/jdk8.toolchains.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF8"?>
+<toolchains>
+    <toolchain>
+        <type>jdk</type>
+        <provides>
+            <version>1.8</version>
+            <vendor>openjdk</vendor>
+        </provides>
+        <configuration>
+            <jdkHome>/usr/local/openjdk-8</jdkHome>
+        </configuration>
+    </toolchain>
+</toolchains>
```

Comment:
This is the only bit of toolchain I needed to get BFD builds workin in the maven container environment.